docs/usage/configuration-options.md

```diff
@@ -276,6 +276,14 @@ With a negation, all branches except those matching the regex will be added to t
 
 Configuring this to `true` means that Renovate will detect and apply the default reviewers rules to PRs (Bitbucket only).
 
+## bbUseDevelopmentBranch
+
+By default, Renovate will use the repositories `main branch`.
```

Comment:
```suggestion
By default, Renovate will use a repository's default branch (typically called `main` or `master`).
```

This is same terminology used by Atlassian here:

![image](https://user-images.githubusercontent.com/6311784/223725741-3ff09777-efca-482b-a539-d6a73bf170f7.png)

Reply:
Yeah, i noticed late last night that their documentation language uses `default branch` but that isn't consistent with what their product uses.  

Attached are some screenshots showing what they use across their Advanced configuration, branching model and branches screens

**Advanced configuration for repository**
![Screenshot 2023-03-08 at 8 57 51 AM](https://user-images.githubusercontent.com/386277/223732559-298ef26c-fbd9-4679-8b71-d5ff17feff7d.png)

**main-based development branches**
![Screenshot 2023-03-08 at 8 58 13 AM](https://user-images.githubusercontent.com/386277/223732615-8db5f677-9e1c-4542-84f1-c84afdb097a4.png)

**git-flow based development branches**
![Screenshot 2023-03-08 at 8 58 26 AM](https://user-images.githubusercontent.com/386277/223732686-4ae21fa6-5837-424a-8824-3857640e1eae.png)

**branching model configuration**
![Screenshot 2023-03-08 at 9 00 47 AM](https://user-images.githubusercontent.com/386277/223732717-6f8d3fdb-adf6-4e64-89b2-009edc68c01f.png)

Reply:
Happy to update the docs to the proposed wording if you still think that makes most sense

Reply:
Hard to tell what to do if they are inconsistent themselves. Seems like main is used more